.ci/docker/ci_commit_pins/nccl-cu12.txt

```diff
@@ -1 +1 @@
-v2.25.1-1
```

Comment:
Ugh, we forgot to update some of the common docker install sh scripts to the latest NCCL version too?

Reply:
@Skylion007 will this fix it? https://github.com/pytorch/pytorch/pull/149778